src/odemis/driver/test/ueye_test.py

```diff
@@ -97,5 +98,63 @@ def setUp(self):
         VirtualTestCam.setUp(self)
 
 
+class TestUEyeCroppingTranspose(VirtualTestCam, unittest.TestCase):
+    """
+    Test setting up the UEye class with a transposed image and a set resolution to force cropping.
+    """
+    camera_type = CLASS
+    camera_kwargs = KWARGS
+
+    @classmethod
+    def setUpClass(cls):
+        if TEST_NOHW:
+            raise unittest.SkipTest('No camera HW present. Skipping tests.')
+
+        # set a fixed max_res smaller than the maximum supported camera resolution
+        cls.camera_kwargs['max_res'] = (1000, 1000)
+        cls.camera_kwargs['transp'] = [2, -1]
+
+        super(TestUEyeCroppingTranspose, cls).setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
+        if TEST_NOHW:
+            return
+        super(TestUEyeCroppingTranspose, cls).tearDownClass()
+
+    def setUp(self):
+        if TEST_NOHW:
+            self.skipTest("No simulator available")
+
+        VirtualTestCam.setUp(self)
```

Comment:
You don't need these 2 overrides if you already have the SkipTest in the setUpClass(), or do you?